engine/src/main/java/com/odysseusinc/arachne/executionengine/execution/r/DockerService.java

```diff
@@ -0,0 +1,166 @@
+package com.odysseusinc.arachne.executionengine.execution.r;
+
+import static com.github.dockerjava.api.model.ResponseItem.ProgressDetail;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import com.github.dockerjava.api.command.PullImageResultCallback;
+import com.github.dockerjava.api.model.Bind;
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.api.model.PullResponseItem;
+import com.github.dockerjava.api.model.Volume;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientImpl;
+import com.github.dockerjava.httpclient5.ApacheDockerHttpClient;
+import com.github.dockerjava.transport.DockerHttpClient;
+import com.odysseusinc.arachne.execution_engine_common.api.v1.dto.AnalysisSyncRequestDTO;
+import com.odysseusinc.arachne.execution_engine_common.api.v1.dto.Stage;
+import com.odysseusinc.arachne.executionengine.config.properties.DockerRegistryProperties;
+import com.odysseusinc.arachne.executionengine.execution.Overseer;
+import com.odysseusinc.datasourcemanager.krblogin.KrbConfig;
+import java.io.File;
+import java.net.URI;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+@Slf4j
+public class DockerService extends RService implements AutoCloseable {
+    private static final String WORKDIR = "/etc/analysis";
+    private final DockerClient client;
+    /**
+     * Directory prefix on HOST where the analysis files are unpacked.
+     */
+    @Value("${analysis.mount}")
```

Comment:
While it is true that this directory does participate in the process of aquiring results from an execution, calling it "analysis results folder" will be misleading. It is a directory on host machine which is connected to a current container under `etc/executions` via docker bind mount (https://docs.docker.com/storage/bind-mounts/) used as a parent directory to create temporary directories for analysis execution.
Thus, the word 'mount' is more important. Since it is difficult to convey the exact meaning in 2-3 words, I have opted to keep it short and put documentation both in javadoc and README.
The current naming is not a hardpoint, if you insist on changing it, please provide a new name and I will update it.